book/website/project-design/sdp/pd-sdp-private-learning.md

```diff
@@ -0,0 +1,51 @@
+
+### Useful resources
+
+- [Individual Risk Tool](https://cpg.doc.ic.ac.uk/individual-risk/) *A useful visualisation of how a few pieces of information can uniquely identify you*
+- [The Algorithmic Foundations
+of Differential Privacy](https://privacytools.seas.harvard.edu/files/privacytools/files/the_algorithmic_foundations_of_differential_privacy_0.pdf) *The foundational paper to the study of differential privacy.*
+- [What is privacy-preserving synthetic data?](https://www.statice.ai/post/what-is-synthetic-data-introduction) *A straightforward introduction to the concept of synthetic data*
+
+## Learning with privacy
+
+Beyond sharing data with other researchers, we can also share our trained models, or make them available as a service: carrying out predictions on data provided by others without the need for them to invest time and resources in training their own systems. However, this sharing can also carry risks for personal privacy. For instance, many ML solutions require users to send personal data to a central server to process, exposing them to the risk of interception or misuse. The model itself may learn sequences from the dataset that we don't wish to be retained, a process referred to as unintended memorization {ref:ps}`carliniSecretSharerEvaluating2019`. This could be particularly harmful when considering models dealing with large amounts of user-created text {cite:ps}`brownWhatDoesIt2022`.
+
+### Federated learning
+
+Federated Learning is a design paradigm in which the users' data never leaves their own devices, with the model itself being broken down into a set of computations that take place on the edge, before updates are sent back to a central coordinator {cite:ps}`kairouzAdvancesOpenProblems2019`.
+
+### Adversarial learning
+
+We can also draw on the experience of research in the field of cross-domain training to teach models to ignore undesirable data by directly controlling the training process {cite:ps}`Coavoux2018`. This can also be extended beyond private attributes to elimination of unwanted biases {cite:ps}`zhangMitigatingUnwantedBiases2018`.
+
+### Differential privacy
+
+Differential privacy has also seen significant use as a technique for preserving privacy during model training, reducing the risk of the model learning individual data points too well by adding small amounts of statistical noise during training {cite:ps}`boulemtafesReviewPrivacypreservingTechniques2020,feyisetanPrivacyUtilitypreservingTextual2020`. 
+
+### Useful resources
+
+- [Privacy in Deep Learning: A Survey](https://cseweb.ucsd.edu/~fmireshg/survey_pvdl_2020.pdf) *A useful brief overview of some extant threats and mitigation strategies.*
+- [Deep learning and differential privacy](https://github.com/frankmcsherry/blog/blob/master/posts/2017-10-27.md) *Frank McSherry's thought provoking blog about the privacy research landscape.*
```

Comment:
You can replace this to get around the inclusive language test (which fails when it finds the word 'master'), but note that the link I've used below is the permalink to the version that exists now (it won't be updated if the file is updated in the future). If that's fine, I suggest you replace the link. If it's not, wait and see what other options present themselves :)

```suggestion
- [Deep learning and differential privacy](https://github.com/frankmcsherry/blog/blob/fdc265de245a82beb38b9a4f28799ef12f556ac1/posts/2017-10-27.md) *Frank McSherry's thought provoking blog about the privacy research landscape.*
```

Reply:
Seems like a bad pattern to replace a dynamic link with a static one that could go stale at any moment. Here's hoping the issue gets addressed upstream.